odoo-wrapper

```diff
@@ -51,5 +51,7 @@ su postgres -c "pg_ctl stop"
 # Generate coverage report, if applicable
 #
 if [ -n "${addons}" ] ; then
-    su odoo -l -c "coverage3 report --show-missing"
+    su odoo -l -c "cd /opt/odoo-addons/; coverage3 report --show-missing"
+    su odoo -l -c "cd /opt/odoo-addons/; coverage3 xml"
+    su odoo -l -c "cd /opt/odoo-addons/; diff-cover coverage.xml"
```

Comment:
This could be:
su odoo -l -c "cd /opt/odoo-addons/ && coverage3 report --show-missing && coverage3 xml && diff-cover coverage.xml"

Each command is only meaningful if the one before it passes.

Reply:
the "coverage3 report --show-missing" bit will generate a report and so will the "diff-cover" command so should I have that on 2 lines?